flaskbb/templates/management/banned_users.html

```diff
@@ -1,5 +1,5 @@
 {% set page_title = _("Banned Users") %}
-{% set active_management_user_nav=True %}
+{% set active = "management.users" %}
```

Comment:
Could we set this on `g` or something and autoload it via a prerequest processor?

Reply:
Do you mean that we set the currently active blueprint on g?

Reply:
Just the endpoint itself, unless request is available in the templates already.

Reply:
``request.endpoint`` is available in the templates. In the conversations plugin I am handling it like this: https://github.com/sh4nks/flaskbb-plugins/blob/master/conversations/conversations/templates/_inject_navlink.html#L2